TMDB.php

```diff
@@ -98,20 +101,20 @@
  * 		Person images		http://api.themoviedb.org/3/person/287/images
  */
 
-include("controller/classes/data/ApiBaseObject.php");
-include("controller/classes/data/Collection.php");
-include("controller/classes/data/Company.php");
-include("controller/classes/data/Episode.php");
-include("controller/classes/data/Genre.php");
-include("controller/classes/data/Movie.php");
-include("controller/classes/data/Person.php");
-include("controller/classes/data/Role.php");
-include("controller/classes/data/Season.php");
-include("controller/classes/data/TVShow.php");
-include("controller/classes/roles/MovieRole.php");
-include("controller/classes/roles/TVShowRole.php");
-include("controller/classes/config/APIConfiguration.php");
-include("controller/classes/config/Configuration.php");
+require ("controller/classes/data/Collection.php");
+require ("controller/classes/data/Company.php");
+require ("controller/classes/data/Episode.php");
+require ("controller/classes/data/Genre.php");
+require ("controller/classes/data/Movie.php");
+require ("controller/classes/data/Person.php");
+require ("controller/classes/data/Role.php");
+require ("controller/classes/data/Season.php");
+require ("controller/classes/data/TVShow.php");
+require ("controller/classes/roles/MovieRole.php");
+require ("controller/classes/roles/TVShowRole.php");
+require ("controller/classes/config/APIConfiguration.php");
+require ("controller/classes/config/Configuration.php");
```

Comment:
If we use the `Composer` support for this repository, why we still include the every required classes manually?